meetup/services/meetup_api_response_parser.py

```diff
@@ -0,0 +1,11 @@
+def parse(api_response):
+  json = api_response.json()
+  parsed_response = {}
+
+  if json:
+    parsed_response['venue'] = json[0].get('venue').get('name')
```

Comment:
A concise helper method might clean this up enough that you could use a dictionary literal here, too, e.g.:

```py
{
  'venue': extract_attribute('venue', 'name'),
  'next_event_topic': extract_attribute('name')
}
```